drivers/sensor/sx9500/sx9500.c

```diff
@@ -135,8 +128,15 @@ int sx9500_init(const struct device *dev)
 	return 0;
 }
 
+static const struct sx9500_config sx9500_config = {
+	.i2c = I2C_DT_SPEC_INST_GET(0),
+#ifdef CONFIG_SX9500_TRIGGER
+	.int_gpio = GPIO_DT_SPEC_INST_GET(0, int_gpios),
+#endif
```

Comment:
for future cleanups: better use `IF_ENABLED` to facilitate conversion to multi-instance later.